exasol/analytics/audit/audit.py

```diff
@@ -0,0 +1,74 @@
+from inspect import cleandoc
+from typing import List
+
+from exasol.analytics.audit.columns import AuditColumns
+from exasol.analytics.query_handler.query.select import (
+    AuditQuery,
+    ModifyQuery,
+    SelectQueryWithColumnDefinition,
+)
+from exasol.analytics.schema import (
+    Column,
+    SchemaName,
+    TableLikeName,
+    TableLikeNameImpl,
+    decimal_column,
+    timestamp_column,
+    varchar_column,
+)
+
+
+class TableDescription:
+    """
+    This class describes an SQL table by its attributes
+    * table (instance of (TableLikeName)) defining the name and optionally the
+      database schema of the SQL table
+    * columns the names and types of the columns of the SQL table
+
+    The class also offers a property `create` which renders the attributes
```

Comment:
here it would be better to use the method name render_create

Reply:
See next push.